integrationtests/src/test/java/io/kroxylicious/proxy/KafkaClusterIT.java

```diff
@@ -28,13 +31,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.time.Duration;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import io.kroxylicious.proxy.testkafkacluster.ContainerBasedKafkaCluster;
+import io.kroxylicious.proxy.testkafkacluster.KafkaCluster;
+import io.kroxylicious.proxy.testkafkacluster.KafkaClusterConfig;
+import io.kroxylicious.proxy.testkafkacluster.KafkaClusterFactory;
+import io.kroxylicious.proxy.testkafkacluster.KeytoolCertificateGenerator;
```

Comment:
this was auto-formatted by the maven plugin

Reply:
@SamBarker is there an issue for that?

Although the code formatter doesn't quite format code to my personal preference I don't mind it. The consistency is good and it is less of a time sink than having formatting rules that require me to change the code by hand. But it would be great if we could have the formatter and intellij be more consistent with each other.

Reply:
There wasn't so I've raised https://github.com/kroxylicious/kroxylicious/issues/105 to get it off my mental to do list.